network/server.go

```diff
@@ -181,6 +182,11 @@ func NewServer(logger hclog.Logger, config *Config) (*Server, error) {
 	logger.Info("LibP2P server running", "addr", AddrInfoToString(srv.AddrInfo()))
 
 	if !config.NoDiscover {
+
+		if config.Chain.Bootnodes != nil && len(config.Chain.Bootnodes) < 2 {
```

Comment:
Nitpick comment. Shall we define a constant for this magic number? And it's good you can write why you needed to set this value to 2.

```go
const (
    MinimumBootnodes = 2 // Why you set MinimumBootnodes to 2
)

...

		if config.Chain.Bootnodes != nil && len(config.Chain.Bootnodes) < MinimumBootnodes {
```

Reply:
Thanks for your suggestion @Kourin1996 , Here's is the commit [link](https://github.com/0xPolygon/polygon-sdk/pull/260/commits/afc4a6e9c20d683dd70fe56885ce552d37ca539b)